### DISCUSSION TITLE
failed to realize "Get Started"

### DISCUSSION THREAD

**Discussion by Justinliuzq** [iTwin/itwinjs-core#3432] [Q&A]
I have followed the "Get Started "(https://developer.bentley.com/tutorials/web-application-quick-start/), but the final website contained nothing
![2)RJBC@@} QK 9S`W$D)TMA](https://user-images.githubusercontent.com/85337086/160552930-89936f42-cd37-461f-8882-8ff47382d32a.png)
![ZFP@VHBZA_1EE9VIKZPGFEK](https://user-images.githubusercontent.com/85337086/160552929-16be4835-9c70-4a57-b81a-8c7b60bc6e98.png)
.

**Comment by Snailya** [comment]
It seems like you are using local area network as you ip address is 172.17.X.X. 
The sample viewer application need to connect to Bentley's Service to authorize and fetch imodel data from imodeHub.
I think it's better to open the develop tool with F12 to see error messages.

**Reply by Justinliuzq** [reply]
According to the console ,I have rebuilt the client ID and imodel ID .But the error still exists.
![AEVC2 7{1 3J7U$)_5OJQEC](https://user-images.githubusercontent.com/85337086/160565244-5e420874-c9de-4f3a-8ebb-c228f5a083d6.png)

**Reply by aruniverse** [reply]
Hi! As the first error suggests, you're missing some credentials. Make sure you save your `.env` file, and restart your client. The second error can be resolved by setting `react-error-overlay` to version `6.0.9`, see [this issue](https://github.com/facebook/create-react-app/issues/11773#issuecomment-995933869)
